microsoft/testsuites/ltp/ltp.py

```diff
@@ -69,6 +69,11 @@ def dependencies(self) -> List[Type[Tool]]:
     def can_install(self) -> bool:
         return True
 
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Arguments can be passed in the construct method, instead of initializing.

Reply:
I did not understand, 
You're suggesting we initialize the value in __init__ ? It is recommended to not override init right?

Reply:
It needs to overrite `__init__` and make sure the super methods is called. It's confusing to pass arguments to both `__init__` and `_initialize`.

Reply:
Updated